vignettes/robust.prioritizr.Rmd

```diff
@@ -173,7 +173,9 @@ p2 <- problem(sim_pu_raster, sim_features_robust_raster) %>%
   add_default_solver(verbose = FALSE)
 
 s2 <- solve(p2, force = TRUE)
-plot(s2)
+soln <- c(s1, s2)
+names(soln) <- c("Non-robust", "Robust")
```

Comment:
consider adding solution to the plot names here to make it obvious? e.g., "Non-robust solution"